n3fit/src/n3fit/layers/x_operations.py

```diff
@@ -1,54 +1,55 @@
 """
     This module contains layers acting on the x-grid input of the NN
 
-    The three operations included are:
+    The two operations included are:
         - ``xDivide``
         - ``xIntegrator``
 
     The names are self-describing. The only subtlety is that they do not act equally
     for all flavours. The choice of flavours on which to act in a different way is given
     as an input argument.
 """
+from typing import List
 
 from n3fit.backends import MetaLayer
 from n3fit.backends import operations as op
 
-BASIS_SIZE=14
+BASIS_SIZE = 14
+
 
 class xDivide(MetaLayer):
     """
-    Divide some PDFs by x
+    Create tensor of either 1/x or ones depending on the flavour,
+    to be used to divide some PDFs by x by multiplying with the result.
 
-    By default it utilizes the 14-flavour FK basis and divides [v, v3, v8]
-    which corresponds to indices (3,4,5) from
+    By default it utilizes the 14-flavour FK basis and divides [v, v3, v8, v15]
+    which corresponds to indices (3, 4, 5, 6) from
     (photon, sigma, g, v, v3, v8, v15, v24, v35, t3, t8, t15, t24, t35)
 
     Parameters:
     -----------
         output_dim: int
             dimension of the pdf
         div_list: list
-            list of indices to be divided by X (by default [3,4,5]; [v, v3, v8]
+            list of indices to be divided by X (by default [3, 4, 5, 6]; [v, v3, v8, v15]
     """
 
-    def __init__(self, output_dim=BASIS_SIZE, div_list=None, **kwargs):
+    def __init__(self, output_dim: int = BASIS_SIZE, div_list: List = None, **kwargs):
```

Comment:
```suggestion
    def __init__(self, output_dim: int = BASIS_SIZE, div_list: List | None = None, **kwargs):
```
Same as above and everywhere else below! A very minor point and hence could be discarded. If you want, however, you could also use `typing.Union`.